README.md

```diff
@@ -1,11 +1,11 @@
-# OpenExplorer
+# OpenENM
 
 [//]: # (Badges)
-[![GitHub Actions Build Status](https://github.com/uibcdf/OpenGNM/workflows/CI/badge.svg)](https://github.com/uibcdf/OpenGNM/actions?query=workflow%3ACI)
```

Comment:
Thanks! This was something we didn't notice before.

Reply:
Thanks!